phoenix-core/src/main/java/org/apache/phoenix/util/SchemaUtil.java

```diff
@@ -1183,6 +1186,22 @@ public static int getIsNullableInt(boolean isNullable) {
 		return isNullable ? ResultSetMetaData.columnNullable : ResultSetMetaData.columnNoNulls;
 	}
 
+	public static int getTimeToLive(PhoenixConnection conn, String physicalName) throws SQLException {
+        byte[] tableQualifier = Bytes.toBytes(physicalName);
+        return getTimeToLive(conn, tableQualifier);
+    }
+
+    public static int getTimeToLive(PhoenixConnection conn, byte[] tableQualifier)
+     throws SQLException {
+        HTableDescriptor td = conn.getQueryServices().getTableDescriptor(tableQualifier);
+        HColumnDescriptor[] cds = td.getColumnFamilies();
+        if (cds.length > 0){
+            return cds[0].getTimeToLive();
+        } else {
+            return HConstants.FOREVER;
```

Comment:
when would this be the case? i.e. #CF==0

Reply:
I couldn't think of one, but decided to code the function defensively.